pkg/oc/admin/policy/reconcile_clusterrolebindings.go

```diff
@@ -261,27 +266,42 @@ func (o *ReconcileClusterRoleBindingsOptions) ReplaceChangedRoleBindings(changed
 		}
 
 		if kapierrors.IsNotFound(err) {
-			createdRoleBinding, err := o.RoleBindingClient.Create(changedRoleBindings[i])
+			roleBinding, err := util.ClusterRoleBindingFromRBAC(changedRoleBindings[i])
+			if err != nil {
+				errs = append(errs, err)
+				continue
+			}
+			createdRoleBinding, err := o.RoleBindingClient.Create(roleBinding)
 			if err != nil {
 				errs = append(errs, err)
 				continue
 			}
 			fmt.Fprintf(o.Out, "clusterrolebinding/%s\n", createdRoleBinding.Name)
 			continue
 		}
+		rbacRoleBinding, err := util.ClusterRoleBindingToRBAC(roleBinding)
+		if err != nil {
+			errs = append(errs, err)
+			continue
+		}
 
 		// RoleRef is immutable, to reset this, we have to delete/recreate
 		if !kapihelper.Semantic.DeepEqual(roleBinding.RoleRef, changedRoleBindings[i].RoleRef) {
-			roleBinding.RoleRef = changedRoleBindings[i].RoleRef
-			roleBinding.Subjects = changedRoleBindings[i].Subjects
+			rbacRoleBinding.RoleRef = changedRoleBindings[i].RoleRef
```

Comment:
@liggitt I do not understand why these 2 lines exist at all.  Maybe they were never supposed to be inside the if statement?